src/pages-and-resources/discussions/DiscussionAppCard.jsx

```diff
@@ -45,28 +45,28 @@ function DiscussionToolOption({
           <div className="d-flex justify-content-center">
             <Image
               height={100}
-              src={forum.logo}
+              src={app.logo}
               alt={intl.formatMessage(messages.toolLogo, {
-                toolName: forum.forumId,
+                toolName: app.id,
```

Comment:
The data layer has a corresponding change that turns "forumId" into two new fields, "id" and "name".  It can be seen in api.js